### PR TITLE
Simplify pipeline

### LINKED ISSUE #6: Update readme
```
# Run the script
python flows/finance-vix.py
```
I am assuming this is something to do after cloning this repository? There is no directory `flows`.

### PR DESCRIPTION
**Pull request by davidgasquez**
Heya! 

Spent some time this morning trying to simplify this core dataset pipeline to some "minimal working state".

Witht these changes, GitHub Actions will run `curl` on a schedule and save the CSV to the data folder.

Feedback appreciated!

### CODE REVIEW

File: .datahub/datapackage.json
Comment (rufuspollock) [inline .datahub/datapackage.json:1]: Yay, removing the old "let's build our own data pipeline system stuff from back in 2017!"

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+on:
```
Comment (rufuspollock) [inline .github/workflows/run.yml:1]: Looks good!

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+all: data
```
Comment (rufuspollock) [inline Makefile:1]: I dream of the day of a better makefile when we aren't doing scripting in bash. But hey it works and its standard 😄

File: datapackage.yaml
Comment (rufuspollock) [inline datapackage.yaml:1]: I prefer yaml too 😉

One thought: if we are going the yaml way why don't we just inline this as frontmatter into the README - that's the direction i've been heading with DataHub ...

wdyt? for now we'll merge anyway ...

**Review by rufuspollock — APPROVED** [review]
Great job 👏 Let's get it merged!

File: Makefile (hunk 2 above)
Comment (davidgasquez) [reply Makefile:1]: There are things like [`just`](https://github.com/casey/just) and others but is hard to beat `make` in terms of availability anywhere you want to run it.

File: datapackage.yaml
Comment (davidgasquez) [reply datapackage.yaml:1]: > why don't we just inline this as frontmatter into the README

Would all the [`frictionless` features](https://framework.frictionlessdata.io/) work with the YAML in the frontmatter (e.g: `frictionless validate github.com/datasets/finance-vix`)?

Comment (rufuspollock) [reply datapackage.yaml:1]: @davidgasquez i'm not sure they would - that is a good point. It would be nice to try and make them do so (or request that).

Comment (davidgasquez) [reply datapackage.yaml:1]: https://github.com/frictionlessdata/frictionless-py/issues/1617

**Pull request merged**